### ISSUE TITLE
Problem  with  the app

### ISSUE THREAD

**Issue by moak** [EQuimper/streamingAppRedux#57]
Hi, I recently forked your app to see how it was and I saw a few problems, here are the points I noticed and want to talk about:
- The bundle.js is committed and used in the app by default, why is that ? 
- Problem with loaders and babel and ES6 features if using a static bundle.js
- The twitter API routes for getting the streams and a specific stream don't work.
- The persistence of the URL doesnt work ?  if you directly go  to http://localhost:3000/games/League%20of%20Legends for example, i get a bank page with nothing.
- There are also problems with the recent refacto with ROOT_URL and URL_ROOT as well as wrong API url

This project is great, well done ! But i feel like it would need a little update would be nice. I created a blank project using create-react-app, imported your files and fixed the bugs I told you about. I didnt create a pull request because my version is using create-react-app so I don't know if you want to swap from your "old" project config to the new official react one. Tell me what you think, i have my app in front of me, it works just perfectly :)

Have a good day.

Maxime

**Comment by EQuimper** [comment]
Thank @moak 
- Yes I have some refactor to do I make this app when I just start learning coding like I think just 3 month in.
- Twitter api ? You mean the twitch one ? Yes I need to fix this one too.
- Persitence not working cause my router is only client no server so. But this is my plan to build a backend with user auth etc.
- Ok let me look at the url, that coming from a pr of someone else so I'm going to check.

**Comment by moak** [comment]
Yes i meant the Twitch API sorry. If you want I can upload my version using the create-react-app official repo. So you can compare both versions ?

**Comment by EQuimper** [comment]
Yes upload it I'm gonna watch this.

If you want we can build backend stuff and make it with more features. I didn't touch it in past week cause I work a lot. But if I got idea of features gonna be happy to put hours on ;)

**Comment by moak** [comment]
Hey! https://github.com/moak/twitchApp  here is the project without warnings or errors. It uses create-react-app and i think it could be a nice "restart" for the project. I also rencently started developing with react/redux and I want to improve my skills so yeah, i am in if you want to add more features.  For  the server, I think it's better to have separated folders for the client and the server. I would put the entire actual project inside a client folder and then use express in a server folder to keep the concerns separated. Then there is a proxy feature in create-react-app to say on which port we want our API to be started. Tell me what you think, if you like it, i make a pull request :)

**Comment by moak** [comment]
up?

**Comment by EQuimper** [comment]
@moak yep perfect make pull request ;)

**Comment by moak** [comment]
Nice :) I will make a pull request really soon.

PS: Are you a teacher ?

**Comment by EQuimper** [comment]
Why you ask this lol ?

**Comment by moak** [comment]
haha I don't know, i checked your repos, and one of them is for "students"
:) I was a student myself two years ago, and i would have loved working
with those technologies.

**Comment by EQuimper** [comment]
I help student in my past online bootcamp

**Comment by moak** [comment]
Hey :) I am back on react :+1:  I did the pull request, everything seems to work smoothly. We will now just have to create a server/client wrapper folder if you want to add a gateway or any server side in the future.
